apps/backend/src/core/events.py

```
@@ -0,0 +1,376 @@
+        """Stop the event processing loop gracefully"""
+        if not self._running:
+            return
+        
+        self._running = False
+        
+        if self._processor_task:
+            self._processor_task.cancel()
+            try:
+                await self._processor_task
+            except asyncio.CancelledError:
+                pass
```

Ruff (0.12.2)

Use `contextlib.suppress(asyncio.CancelledError)` instead of `try`-`except`-`pass`

Replace with `contextlib.suppress(asyncio.CancelledError)`

(SIM105)